packages/dashboard/src/views/Holdings.js

```diff
@@ -190,6 +193,53 @@ const Holdings = ({
   // todo move this to only update on account change
   const accountSmallStats = renderSmallStats()
 
+  const renderTableData = () => {
+    // check if we have account  data
+    if (historicQuotes['1d'] && historicQuotes['all']) {
+      const positions = selectedAccount.positions
+      const currDayData = historicQuotes['1d']
+      const currDayResults = currDayData.results[currDayData.results.length - 1]
+      const portfolioValue = currDayResults.value.amount
+
+      return positions.map(position => (
+        <tr key={position.id}>
+          <td>{position.stock.symbol}</td>
+          <td>{position.quantity}</td>
+          <td>
+            {'$' + (position.book_value.amount / position.quantity).toFixed(2)}
+          </td>
+          <td>{'$' + position.quote.amount}</td>
+          <td>
+            {'$' + (position.quote.amount * position.quantity).toFixed(2)}
+          </td>
+          <td>
+            {'$ ' +
+              (
+                position.quote.amount * position.quantity -
+                position.book_value.amount
+              ).toFixed(2)}
+          </td>
+          <td>
+            {(
+              ((position.quote.amount * position.quantity -
+                position.book_value.amount) /
+                position.book_value.amount) *
+              100
+            ).toFixed(2) + '%'}{' '}
```

Comment:
a good way of doing string interpolation is using template literals https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
just lets you sub variables and strings in easily ``const string = `value is: ${value}CAD` ``
just for future this is usually easier

Reply:
Roger that